web/elm/src/SideBar/Pipeline.elm

```diff
@@ -20,6 +21,8 @@ pipeline :
     { a
         | hovered : HoverState.HoverState
         , currentPipeline : Maybe (PipelineScoped b)
+        , favoritedPipelines : Set Int
+        , isFavoritesSection : Bool
```

Comment:
It feels a bit weird to add this property here when the parameter's name is `session` -- the section of the sidebar being rendered by this function is not actually a property of the `session`. Maybe this could be a separate parameter, or else this parameter could instead have a more generic name like `params`?